homeassistant/components/apple_tv.py

```diff
@@ -91,7 +91,7 @@ def configuration_callback(callback_data):
         hass.async_add_job(configurator.request_done, instance)
 
     instance = configurator.request_config(
-        hass, 'Apple TV Authentication', configuration_callback,
+        'Apple TV Authentication', configuration_callback,
```

Comment:
My personal preference would be to hold off the Automatic changes to the next release. I know people are waiting for a fix, but I don't like the idea of introducing a new breaking change with a point release. It's available in dev if people don't want to wait.

That being said I'd also understand the argument that it's already broken as-is. If we do end up picking this I'd just request we get polymer master built-in as well to include the configurator update.